css/styles.css

```diff
@@ -0,0 +1,3 @@
+body header{
```

Comment:
you can add  more height in pixels to look header better

Reply:
ALL other thing is fine font style will be a good option for header

Reply:
Changes are done, please check them